cves/kernel/CVE-2021-28951.yml

```diff
@@ -75,7 +78,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [1941769]
```

Comment:
Maybe add bugs [1941770](https://bugzilla.redhat.com/show_bug.cgi?id=1941770) and [1943398](https://bugzilla.redhat.com/show_bug.cgi?id=1943398) that are also involved in this vulnerability